main/openrc/modloop.initd

```diff
@@ -10,7 +10,7 @@ depend() {
 
 # read kernel options
 init_KOPT() {
-	eval set -- $(cat /proc/cmdline 2>/dev/null)
+	eval set -- $(cat /proc/cmdline 2>/dev/null|sed -e 's/(loop)//g')
```

Comment:
@ncopa Why is eval used here first place? Isn't `set` enough? Could we just drop the `eval` instead of adding sed kludge?